util/unifyfs/src/unifyfs-rm.c

```diff
@@ -244,6 +248,99 @@ static int wait_server_initialization(unifyfs_resource_t* resource,
     return ret;
 }
 
+/**
+ * @brief wait until data stage operation finishes
+ *
+ * @param resource
+ * @param args
+ *
+ * @return
+ */
```

Comment:
What is this comment for?

Reply:
I think it was meant to be attached to a function farther down.  Moved.